elixir/apps/web/test/web/controllers/sign_in_controller.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Web.SignInControllerTest do
+  use Web.ConnCase, async: true
+
+  setup do
+    Domain.Config.put_env_override(:outbound_email_adapter_configured?, true)
+    account = Fixtures.Accounts.create_account()
+
+    {:ok, account: account}
+  end
+
+  describe "success/2" do
+    test "renders success page on proper cookie values", %{conn: conn} do
+      provider = Fixtures.Auth.create_email_provider(account: account)
+      actor = Fixtures.Actors.create_actor(type: :account_admin_user, account: account)
+      identity = Fixtures.Auth.create_identity(account: account, provider: provider, actor: actor)
+
+      conn_with_cookies =
+        conn
+        |> put_client_auth_state(account, provider, identity)
+        |> get(~p"/#{account}/sign_in/success")
+
+      html = response(conn_with_cookies, 200)
+
+      assert html =~ "Sign in successful"
+      assert html =~ "close this window"
```

Comment:
We should also test that JS redirect is at least scheduled and that all required parameters will be part of its URL.

Reply:
There is no JS redirect.  The redirect is happening using a `<meta>` tag now.  I had to use the `<meta>` tag in our initial redirect from an external IDP, so rather than have it do a redirect to a page that did another redirect (using JS), I just cut out that middle redirect and had it go directly to the `/sign_in/deep_link` path.

Reply:
Got it. Let's at least use Floki to read that redirect tag, make sure it's there and that redirect URI is what we want it to be

Reply:
Updated.